tests/integration/from-babel-eslint.spec.js

```diff
@@ -0,0 +1,1738 @@
+"use strict";
+
+const unpad = require("dedent");
+
+const utils = require("./utils");
+const verifyAndAssertMessages = utils.verifyAndAssertMessages;
+
+describe("Non-regression tests from babel-eslint", () => {
```

Comment:
Do we need to copy-paste `babel-eslint`'s license here since the tests are from `babel-eslint`?

Reply:
Yes, definitely

Reply:
Done